.github/workflows/test-all.yml

```diff
@@ -0,0 +1,60 @@
+name: Test browsers
+
+on:
+    pull_request:
```

Comment:
So this needs to be replaced with `push` before merge AFAIU? I think we can already change it since we are almost done :+1:

Reply:
Adjusted trigger conditions